### ISSUE TITLE
Electrolytic Separator won't accept heavy water.

### ISSUE THREAD

**Issue by 15kone** [mekanism/Mekanism#7845]
### Issue description

I'm trying to make deuterium for my fission reactor, but my electorlytic separator will not accept the heavy water I need to pump into it in order to create it.

### Steps to reproduce

1. Pump out heavy water into a tank
2. Try to get it into a electrolytic separator
3. Fail

### Minecraft version

1.19.2 (Latest)

### Forge version

43.2.14

### Mekanism version

Current Dev Branch (Manually Compiled)

### Other relevant versions

ATM 8, playing on a server with friends

**Comment by pupnewfster** [comment]
How are you trying to get it out of the tank?

**Comment by 15kone** [comment]
> How are you trying to get it out of the tank?

Mechanical pipe from mekanism

**Comment by pupnewfster** [comment]
Is the pipe set to pull? And is the separator configured to accept fluids from the side you are inserting into?

**Comment by 15kone** [comment]
Yes, I thoroughly checked everything, and it just wasn't pulling. For some reason a server restart fixed the issue but it was weird nontheless.

**Comment by Lavendellothar** [comment]
yeah i hav eth esame problem, pipes are full with heavy water but i cant get them into the separator

**Comment by ekconcordian** [comment]
This still appears to be a problem. Might be a multiplayer server issue. I'm also playing ATM 8 with friends. Both an array of electrolytic separators in a Compact Machines dimension and a PRC in the overworld seem to be glitched and refusing input. Server restarts seem to occasionally fix the issue - the PRC fixed itself after a few restarts, while the electrolytic separators are not fixing themselves yet. PRC set up to accept plutonium, and doesn't: 
![image](https://github.com/mekanism/Mekanism/assets/9299352/4762ab97-9995-4492-acf3-26a261404cfc)
![image](https://github.com/mekanism/Mekanism/assets/9299352/7c6ddd0a-72d2-4a20-bfa6-3f4f78f6a632)

And Electrolytic Separators not accepting heavy water:
![image](https://github.com/mekanism/Mekanism/assets/9299352/20db9629-b985-4dea-a40d-bbb5b9af2912)
![image](https://github.com/mekanism/Mekanism/assets/9299352/f88d2624-ebe6-4751-b9c5-a1bf92541c72)

The sides are 100% set correctly, I've confirmed that the fluid is in the pipes. Replacing the machine entirely doesn't fix it. Removing and replacing the upgrades doesn't fix it. Completely removing the mechanical pipes and replacing them doesn't work either. Making sure everything is contained into one chunk didn't change the behavior.

Several interesting things to note: 
1. The PRC not accepting plutonium was an issue that happened immediately when I placed it down. The Electrolytic Separators not accepting heavy water worked at first but broke after a few days of use. It was working when I logged on this morning, and stopped working seemingly randomly this evening - no server restart occurred between.
2. I used Pipez fluid pipes to move the heavy water into the separators at first, those stopped working and when I tried to replace them with mechanical pipes, those didn't work either. It doesn't seem to be related to the pipes directly. 
3. Interestingly, I had to force the mechanical pipes to pull from the quantum entangloporter, even though all sides for fluid were set to output & eject on. That's the only way I could pull the heavy water from it after replacing the Pipez fluid pipes.
4. I can gauge drop / use a bucket to grab heavy water out of my AE2 system, but upon attempting to place either into the red input slot for fluid into an electrolytic separator, it immediately pops back onto my cursor, so I can't manually place fluid into the machines either. Largely seems machine-related. This is not client-side as all other players on the server report the same issue, and relogging / restarting game doesn't fix the issue.
